build/references/cryptographic-primitives.md

```diff
@@ -62,6 +62,43 @@ Message Signature: `0xb52aa0535c5c48268d843bd65395623d2462016325a86f09420c81f142
 
 Morty was never seen again.
 
+## Signed Messages
+
+A standard for interoperable generic signed messages based on the Bitcoin Script format and Ethereum format.
+
+```txt
+sign(sha256(length(prefix) . prefix . length(message) . message))
+```
+
+The prefix is simply the string '\x1AAvalanche Signed Message:\n', where 0x1A is the length of the prefix text and `length(message)` is an int of the message size.
+
+### Gantt Signed Message Specification
+
+```text
++---------------+-----------+------------------------------+
+| prefix        : [26]byte  |                     26 bytes |
++---------------+-----------+------------------------------+
+| messageLength : int       |                      4 bytes |
++---------------+-----------+------------------------------+
+| message       : []byte    |          size(message) bytes |
++---------------+-----------+------------------------------+
+                            |       26 + 4 + size(message) |
+                            +------------------------------+
+```
+
+### Example
```

Comment:
This is an example of the pre-image (the thing that will be hashed and then signed) right? For clarity, can we also include the result after hashing and signing it?